src/node-set.js

```diff
@@ -0,0 +1,19 @@
+const TermSet = require('@rdfjs/term-set')
+
+class NodeSet extends TermSet {
+  addAll (nodes) {
+    for (const node of nodes) {
+      this.add(node)
+    }
+  }
+
+  toArray () {
+    return [...this]
+  }
+
+  toString () {
+    return `NodeSet(${this.size()}): [${this.toArray().join(', ')}]`
+  }
+}
+
+module.exports = NodeSet
```

Comment:
Why not completely replace `NodeSet` with `term-set`? `[...]` would be even shorter than `.toArray`. The other methods are longer to inline them, so it depends on the usage.

Reply:
That's what I first started to do but the code got messy when I started replacing `addAll`. So I thought I would do it that way.

Any change we can add something like `addAll` on `TermSet`?

Reply:
I removed `toString` and `toArray`. I'm going to keep `addAll` for now if you don't mind.

Reply:
OK, I left a comment in the `TermSet` package. I think like you implemented it, it's the best option right now.